mii/grpc_related/modelresponse_server.py

```diff
@@ -44,120 +44,48 @@ def _get_model_time(self, model, sum_times=False):
             model_time = -1
         return model_time
 
-    def GeneratorReply(self, request, context):
-        query_kwargs = self._unpack_proto_query_kwargs(request.query_kwargs)
+    def _run_inference(self, method_name, request_proto):
+        if method_name not in self.method_name_to_task:
+            raise ValueError(f"unknown method: {method_name}")
+
+        task = self.method_name_to_task[method_name]
+        if task not in GRPC_METHOD_TABLE:
+            raise ValueError(f"unknown task: {task}")
 
-        # unpack grpc list into py-list
-        request = [r for r in request.request]
+        conv_funcs = GRPC_METHOD_TABLE[task]
```

Comment:
Can we get a better name for this variable? I think `conv` means `conversion`, but it's unclear.

Reply:
What about simply `conversions`?

Reply:
`conversions` or `conversions_funcs` would work